Parser/pegen/pegen.c

```diff
@@ -1099,6 +1190,11 @@ _PyPegen_run_parser_from_string(const char *str, int start_rule, PyObject *filen
     mod_ty result = NULL;
 
     int parser_flags = compute_parser_flags(flags);
+
+    if (parser_flags & PyPARSE_TYPE_COMMENTS) {
+        tok->type_comments = 1;
```

Comment:
What about getting rid of the `if` here?
```suggestion
        tok->type_comments = (parser_flags & PyPARSER_TPYE_COMMENTS) > 0;
```